manim/mobject/mobject.py

```diff
@@ -43,8 +43,10 @@ class Mobject(Container):
 
     Attributes
     ----------
-    submobjects : :class:`list`
+    submobjects : List[:class:`Mobject`]
```

Comment:
If it is fine can you do [typing](https://docs.python.org/3/library/typing.html)?

Reply:
What exactly do you mean? Adding `-> type` to the function definitions?

Reply:
> Adding -> type to the function definitions?

yeah, picked the wrong line to quote that sadly

Reply:
Of course I could do that, but I don't think that needs to be part of this PR, does it?